microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -1,376 +1,556 @@
+In general, the extension cannot interfere with the normal functioning of the browser or operating system.
 
-#### 1.1.3 Functionality
 
-Your extension must be fully functional.
+###### 1.1.7 Obfuscation
 
-#### 1.1.4 Search and Discovery
+Extensions with obfuscated code aren't allowed.  This includes code within your extension package, as well as any external code or resource fetched from the web.  You may be asked to refactor parts of your code, if it is not reviewable.
 
-Search terms may not exceed seven unique terms and should be relevant to your extension.
 
-#### 1.1.5 Provide appropriate details
+###### 1.1.8 Altering browser settings
 
-You must provide distinct and informative details about your extension and the functionality in listing (metadata) for your extension.  Your extension must provide a valuable and quality user experience.  Your extension must also have an active presence in Microsoft Edge Add-ons.
+Your extension must not, without appropriate user consent, alter, or appear to alter, browser functionality or settings including, but not limited to: the address bar search provider and suggestions, the start or home page, the new tab page, and adding or removing favorites.
 
-#### 1.1.6 Stability and Performance
+Any changes made to the device settings must be done with the user's knowledge and consent, and must be easily reversible by the user.
 
-Your extension must not negatively impact the performance or stability of Microsoft Edge.
+Any alteration of browser settings, such as new tabs and search engines, should be mentioned in the listing or description, and such alterations should use standard APIs to achieve that functionality.  They must not override default settings without user permission.
 
-#### 1.1.7 Obfuscation
 
-Extensions with obfuscated code aren't allowed.  This includes code within your extension package as well as any external code or resource fetched from the web.  You may be asked to refactor parts of your code if it is not reviewable.
+<!-- ------------------------------ -->
+#### 1.2 Security
 
-#### 1.1.8 Altering Browser Settings
+Extensions must declare all permissions that are required for their functionality in the manifest.  Your extension must only request those permissions that are essential for functioning.
 
-Your extension must not, without appropriate user consent, alter, or appear to alter, browser functionality or settings including, but not limited to: the address bar search provider and suggestions, the start or home page, the new tab page, and adding or removing favorite.
+Any [declarativeNetRequest](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest) rules must be clearly declared within the extension, and should not be imported remotely unless necessary for functionality.
```

Comment:
remove the link from mozilla. Do not add any link here

Reply:
Any `declarativeNetRequest` rules must be clearly declared within